### ISSUE TITLE
Common gradient leak

### ISSUE THREAD

**Issue by jaromiru** [elumixor/DRL#1]
There is a gradient leak on line:
https://github.com/elumixor/DRL/blob/f39bf77da281f95e95bc4024d08605e6eb072f9e/A2C.py#L73

It requires `advantage.detach()` (see the policy gradient for details, e.g., in https://arxiv.org/abs/1602.01783)
